workspacer.sln

```diff
@@ -35,6 +35,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "workspacer.TitleBar", "src\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "workspacer.TitleBar.Tests", "src\workspacer.TitleBar.Tests\workspacer.TitleBar.Tests.csproj", "{E4E357BA-0154-4266-A816-B765208E810D}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "workspacer.Sound", "src\workspacer.Sound\workspacer.Sound.csproj", "{80D9E2E0-5A4F-4DD1-AB21-7535DF162A75}"
+EndProject
```

Comment:
You've added a new project. Great!

Do you think any of the new classes in this project would benefit from tests to ensure the adhere to defined behaviour?

I hate seeing new code added without *any* tests :crying_cat_face:

Reply:
Yes definitely, I'll add tests for the situations now that I know this functionality is wanted in main workspacer